src/pages/_app.tsx

```diff
@@ -4,14 +4,46 @@ import type { AppProps } from 'next/app'
 import { useRouter } from 'next/router'
 import type { ReactNode } from 'react'
 import { config } from '@fortawesome/fontawesome-svg-core'
-
 import '@fortawesome/fontawesome-svg-core/styles.css'
 import 'styles/index.scss'
 import '../../public/vendor/fontawesome-pro-5.15.1-web/css/all.min.css'
-
 import '../initIcons'
-import DefaultLayout from 'layout/MVP/DefaultLayout/DefaultLayout'
+import { ApolloClient, InMemoryCache, ApolloProvider } from '@apollo/client'
+import { persistCache, LocalStorageWrapper } from 'apollo3-cache-persist'
+import { GET_COLLECTIONS } from 'operations/queries/getCollections'
+import { localResolvers } from 'operations/resolvers'
 import { BetaContextProvider } from 'stores/betaContext'
+import DefaultLayout from 'layout/MVP/DefaultLayout/DefaultLayout'
+
+/* Begin ApolloClient Setup */
+
+// Initialize cache
+const initCache = () => {
+  const cache = new InMemoryCache()
+  cache.writeQuery({
+    query: GET_COLLECTIONS,
+    data: {
+      collections: [],
+    },
+  })
+
+  // If we have access to the browser, persist the cache in local storage
+  if (typeof window !== 'undefined') {
+    persistCache({
+      cache,
+      storage: new LocalStorageWrapper(window.localStorage),
+    })
+  }
+  return cache
+}
+
+// Set up client with persisted cache and local resolvers
+const client = new ApolloClient({
+  cache: initCache(),
+  resolvers: localResolvers,
+})
+
+/* End ApolloClient setup */
```

Comment:
☝️  Can we move the above setup to a separate file (like `src/stores/apolloClient.ts`) that exports the `client` object?